capa/ida/plugin/model.py

```diff
@@ -488,13 +488,17 @@ def capa_doc_feature_to_display(self, feature):
 
         @param feature: capa feature read from doc
         """
-        if feature[feature["type"]]:
+        key = feature["type"]
+        value = feature[feature["type"]]
+        if value:
+            if key == "string":
+                value = '"%s"' % value
```

Comment:
see `string` comments.

Reply:
use escape helper function (applies to all similar locations)?